packages/daisyui/src/templates/FieldTemplate/FieldTemplate.tsx

```diff
@@ -36,6 +36,7 @@ export default function FieldTemplate<
     displayLabel,
     classNames,
     // Destructure props we don't want to pass to div
+    description,
```

Comment:
nice subtle fix, the only snapshot changes are related to this (so the ID Schema replacement is clean! 🙌 )